posthog/settings.py

```diff
@@ -85,7 +85,8 @@ def get_env(key):
     'rest_framework',
     'loginas',
     'corsheaders',
-    'social_django'
+    'social_django',
+    'debug_toolbar'
```

Comment:
b/c of the below comment, this package won't always be here. Could do something like
```python
try:
    import debug_toolbar
    INSTALLED_APPS.append('debug_toolbar')
    MIDDLEWARE.append('debug_toolbar.middleware.DebugToolbarMiddleware')
except ImportError:
    pass